docs/installation/config/index.rst

```diff
@@ -6,4 +6,6 @@ Configuration
 .. toctree::
    :maxdepth: 1
 
-   env_config
\ No newline at end of file
+   openorganisatie_config
+   env_config
```

Comment:
setup config also has a directive that generates docs for all steps. See
https://open-product.readthedocs.io/en/stable/installation/config/openproduct_config_cli.html#

https://github.com/maykinmedia/open-product/blob/master/docs/installation/config/openproduct_config_cli.rst